### ISSUE TITLE
Icon example is missing alternative text

### ISSUE THREAD

**Issue by mfairchild365** [angular/material2#15013]
#### What is the expected behavior?
Informative icons must convey their meaning to users of screen reader software.

#### What is the current behavior?
The informative icon in the example does not convey its meaning to users to screen reader software.

#### What are the steps to reproduce?
1. Using either of the following screen readers
   - NVDA 2018.4.1 + Firefox 64
   - VoiceOver for IOS (12.1.3) + Safari
2. Open the [icon example](https://material.angular.io/components/icon/examples)
3. Press the down arrow key to have the screen reader read the next item until you reach the icon in the "SVG icons" example.
4. Observe that the icon is not announced

#### Which versions of Angular, Material, OS, TypeScript, browsers are affected?
- Angular Material Example 7.2.2 located at https://material.angular.io/components/
- NVDA 2018.4.1 + FireFox 64
- VoiceOver for IOS (12.1.3) + Safari (12.1.3)

#### Is there anything else we should know?
The documentation for icons states the following:

> When the presence of an icon communicates some information to the user whether as an indicator or by being inlined into a block of text, that information must also be made available to screen-readers

Because this is the only example of an icon, it should be considered to 'communicate information' as described above. Without alternative text for the icon, screen reader users will not be aware that an icon exists on the page.

It might be prudent to add examples for each of the 3 types of icons discussed in the documention: decorative, interactive, and informative.
